scripts/openzfs-merge.sh

```diff
@@ -103,6 +103,18 @@ generate_desc() {
 	USER_MAIL=$(git config user.email)
 	OPENZFS_COMMIT_AUTHOR=$(git log --format="%aN <%aE>" --remotes=openzfs/master -n 1 $OPENZFS_COMMIT)
 	sed -i '/^$/d' "$TMP_FILE"
+
+	# handle github keywords
+	sed -i '/^closes #/Id' "$TMP_FILE"
+	sed -i '/^close #/Id' "$TMP_FILE"
+	sed -i '/^closed #/Id' "$TMP_FILE"
+	sed -i '/^fix #/Id' "$TMP_FILE"
+	sed -i '/^fixes #/Id' "$TMP_FILE"
+	sed -i '/^fixed #/Id' "$TMP_FILE"
+	sed -i '/^resolve #/Id' "$TMP_FILE"
+	sed -i '/^resolves #/Id' "$TMP_FILE"
+	sed -i '/^resolved #/Id' "$TMP_FILE"
```

Comment:
I went ahead and broke it down into 3 lines, one for each main keyword. If you wouldn't mind giving it one more look over.